ytbdownloader.py

```diff
@@ -1,22 +1,40 @@
 from pytube import YouTube 
 from pytube import Playlist 
 import json
+import argparse
 
-def open_json():
+def get_arg():
+    parser = argparse.ArgumentParser(description='Download videos from Youtube')
+    parser.add_argument(
+        '-i', '--input', 
+        type=str, 
+        help='input name of "*.json" file (default is "/config.json") ', # default не по ТЗ, но я хотел попробывать.. сработает или нет
+
+        default='config.json' #не по ТЗ, но я хотел попробывать.. сработает или нет 
+        )
+    args = parser.parse_args()
+
+    return args
+
+def open_json(args):
 # пытается найти и открыть файл config.json 
+
+    if args.input[-5::] != '.json': #теперь можно не писать расширение *.json (не по ТЗ, но я хотел попробывать.. сработает или нет)
+        args.input += '.json'
+
```

Comment:
I think this code is redundant. We do not need to guess file extensions. Please remove these changes related to adding .json

Reply:
I fixed it